tests/hcaptcha.spec.js

```diff
@@ -544,5 +531,112 @@ describe("hCaptcha", () => {
             const script = document.querySelector("head > script");
             expect(script.async).toBeTruthy();
         });
+
+        describe("scriptLocation", () => {
+
+
+             it("validate src without", () => {
```

Comment:
how about "should be document.head by default"?

Reply:
Moved this test, to this area. Just updated it though here [e71a40d](https://github.com/hCaptcha/react-hcaptcha/pull/182/commits/e71a40df0f0d823d608e055f4ff7d16fc6f308e8)